modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -113,35 +113,35 @@ $ ccoctl aws create-iam-roles --name=<name> --region=<aws_region> --credentials-
 .Example
 [source,terminal]
 ----
-$ ccoctl aws create-iam-roles --name=my-aws-efs --credentials-requests-dir= credrequests --identity-provider-arn=arn:aws:iam::123456789012:oidc-provider/example.cloudfront.net/<cluster-ID>
+$ ccoctl aws create-iam-roles --name=my-amazon-efs --credentials-requests-dir= credrequests --identity-provider-arn=arn:aws:iam::123456789012:oidc-provider/example.cloudfront.net/<cluster-ID>
```

Comment:
Don't suggest to update these steps, some resources name should not be updated.

Reply:
I suggest not update in the modules/osd-persistent-storage-csi-efs-sts.adoc

Reply:
@duanwei33 As suggested above, I revoked all the updates in the modules/osd-persistent-storage-csi-efs-sts.adoc.

Cc @codymant